themes/default/content/case-studies/credijusto.md

```diff
@@ -105,6 +105,19 @@ Pulumi made network configurations extremely simple and easy to replicate across
 environments at Credijusto. Below is an example of how Credijusto used Pulumi to configure
 multiple subnets per availability zone and environment.
 
+In addition to using EKS with Cluster Autoscaler to dynamically scale their Kubernetes
+clusters, the Credijusto team also took advantage of the
+[Amazon Relational Database Service](https://aws.amazon.com/rds/) (RDS)
+and [Amazon Aurora](https://aws.amazon.com/rds/aurora/) to enable the database for
+each application to scale with demand.
+
+> "Our database setup relies on a dedicated Amazon Aurora cluster for each product. Everything
+> is configured by Pulumi, by a custom resource that we built to create and manage all related
+> resources (credentials, Option Groups, Parameter Groups, Subnets, Security Groups, Policies, etc.).
+> This ensures a standard configuration across all of our databases."
+>
+> &mdash; Fernando Carlietti, Lead DevOps Engineer, Credijusto.
+
 <img class="block mx-auto md:max-w-4xl my-8" src="/images/case-studies/credijusto-architecture.png" alt="Credijusto architecture">
```

Comment:
@zchase can you replace with this image (attached)?
<img width="960" alt="Credijusto - Architecture" src="https://user-images.githubusercontent.com/59850911/115785829-ffaa5000-a38d-11eb-8b5d-7264ab039ffc.png">

Reply:
👍 Yeah not sure why it got lost when I updated it the first time.